docs/source/quickstart/model_config.md

```diff
@@ -87,3 +94,15 @@ lr_schedule:
   learning_rate_decay_steps: 100000   #default value is 100000
   learning_rate_decay: 0.96              #default value is 0.96
 ```
+
+
+reduce_lr_on_plateau Learning Rate Schedule
```

Comment:
Thanks for adding this here, very practical. 
Also, thanks for being concise and to the point.